app/src/main/java/com/google/maps/android/compose/MapSampleActivity.kt

```diff
@@ -191,6 +196,11 @@ fun GoogleMapView(
         ) {
             Text(text = "RESET MAP", style = MaterialTheme.typography.body1)
         }
+        Button(
```

Comment:
Not sure if this is intentional, but this button is a different style than the others due to Button colors and Text style. It's also missing the 4.dp of padding:

![image](https://user-images.githubusercontent.com/928045/167163253-fe385fa9-3b69-446c-9fbb-f0b3c58dc6ff.png)

Reply:
Will also align this in the same Row as "RESET MAP"